tests/spec/fs.unlink.spec.js

```diff
@@ -104,3 +104,22 @@ describe('fs.unlink', function() {
     });
   });
 });
+
+/**
+ * fsPromises tests
+ */
+describe('fsPromises.stat', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+  it('should return an error if trying to delete a file that does not exist', function(done) {
```

Comment:
To make code more promise-friendly, please remove `done` from the argument

Reply:
thank you for your suggestion :)